src/SFA.DAS.Payments.RequiredPayments.ServiceFabric/packages.config

```diff
@@ -1,4 +1,4 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Would not expect to see a packages.config file for a .net6 app, is this file required?